### PR TITLE
Testing yourtrip

### PR DESCRIPTION
**Pull request by Shazeen15**

### CODE REVIEW

**Review by minachuong — COMMENTED** [review]
Noticing a lot of `.DS_Store` files getting created. To avoid this, `.DS_Store` can be added to `.gitignore` file so that they don't get tracked

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.4.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "history": "^5.0.0",
```
Comment (zan-rosenthal) [inline package.json]: You can remove this since we didn't end up using it.

File: app/javascript/components/pages/__test__/YourTrips.test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import Enzyme, { shallow, render } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { StaticRouter as Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+
+import YourTrips from "../YourTrips";
+
+const history = createMemoryHistory();
```
Comment (zan-rosenthal) [inline app/javascript/components/pages/__test__/YourTrips.test.js]: You can remove this and where it gets imported since we didn't end up using it.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]
Can you add this to your git ignore: `**/.DS_Store`?

**Review by zan-rosenthal — APPROVED** [review]

**Pull request merged**